src/platform/plugins/shared/navigation/public/README.md

```diff
@@ -0,0 +1,143 @@
+# Kibana Navigation Plugin
+
+## Overview
+
+The Navigation plugin provides reusable UI components and services for navigation within Kibana. It serves as a bridge between the core navigation services and the UI layer, offering standardized components that can be used across different parts of the application.
+
+## Plugin Start Contract
+
+The Navigation plugin's start contract provides the following:
+
+```typescript
+interface NavigationPublicStart {
+  ui: {
```

Comment:
I think this highlights the problem with this plugin. it is about two different navigations unrelated to each other. chrome and TopNav 🤷 

maybe we can get rid of some of it and move navigation stuff into core/chrome